wormhole-connect/package.json

```diff
@@ -17,6 +17,7 @@
     "dist"
   ],
   "dependencies": {
+    "@artursapek/vite-plugin-checker": "^0.7.3",
```

Comment:
a fork which contains this fix https://github.com/fi3ework/vite-plugin-checker/pull/389